alembic/versions/34a3fed26733_datetime_type_annotation.py

```diff
@@ -0,0 +1,30 @@
+"""Datetime type annotation

```

Comment:
Since this revision is empty, we shouldn't add it. It was just a test to make sure no db changes are needed.

Reply:
There is some value in retaining it as a marker that no db changes are needed.